include/sys/fs/zfs.h

```diff
@@ -1223,6 +1225,8 @@ typedef enum zfs_ioc {
 	ZFS_IOC_POOL_CHECKPOINT,		/* 0x5a4d */
 	ZFS_IOC_POOL_DISCARD_CHECKPOINT,	/* 0x5a4e */
 	ZFS_IOC_POOL_INITIALIZE,		/* 0x5a4f */
+	ZFS_IOC_GET_BOOKMARK_PROPS,		/* 0x5a50 */
+	ZFS_IOC_REDACT,				/* 0x5a51 */
```

Comment:
Speaking of conflicts, if possible can we shift these new ioctl's down to `0x5a51` and `0x5a52`.  The outstanding TRIM PR which is expected to be merged soon is using `0x5a50`.